src/sentry_utils.c

```diff
@@ -409,3 +422,54 @@ sentry__iso8601_to_msec(const char *iso)
 
     return (uint64_t)time * 1000 + msec;
 }
+
+#ifdef SENTRY_PLATFORM_WINDOWS
+#    define sentry__locale_t _locale_t
+#else
+#    define sentry__locale_t locale_t
+#endif
+
+static sentry__locale_t
+sentry__c_locale()
+{
+    static bool c_locale_initialized = false;
+    static sentry__locale_t c_locale;
+    if (!c_locale_initialized) {
+        c_locale_initialized = true;
+#ifdef SENTRY_PLATFORM_WINDOWS
+        c_locale = _create_locale(LC_ALL, "C");
+#else
+        c_locale = newlocale(LC_ALL_MASK, "C", NULL);
```

Comment:
Not that it matters much, but the third argument is an integer not a pointer. So should probably be `0`.